english/pokemon-game.md

```diff
@@ -0,0 +1,4 @@
+Pokemon Go is a free-to-play location-based augmented 
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.